### ISSUE TITLE
Contrast of the text in the beta banner is not adequate

### ISSUE THREAD

**Issue by LieutenantRoger** [topcoder-platform/micro-frontends-challenges-app#219]
**Summary:**
Contrast of the text in the beta banner does not comply with WCAG standards

**Steps to Reproduce:**

1. Open https://platform.topcoder.com/earn/find/challenges
2. Open the banner on the top

**Actual Result:**
The contrast between the background and the text is lover than the minimum standard.

**Expected Result:**
The contrast should be higher

**Environment Details:**

***Devices(s):***
MacBook Pro
***Operating System and version:***
MacOS Catalina
***Browser and version:***
Safari 14.0.3
**Screen recording or Screen snapshot:**
![Screenshot_2021-10-29_at_16.45.25](/uploads/00761b34b3ede41a53155b9be1248d5c/Screenshot_2021-10-29_at_16.45.25.png)
